src/braket/circuits/circuit.py

```diff
@@ -643,6 +657,69 @@ def add_verbatim_box(
             self._has_compiler_directives = True
         return self
 
+    def measure(self, target_qubits: Optional[np.ndarray] | Optional[int] = None) -> Circuit:
+        """
+        Add a `measure` operator to `self` ensuring only the target qubits are measured.
+
+        Args:
+            target_qubits (Optional[ndarray] | Optional[int]): target qubits to measure.
+                Default=None
+
+        Returns:
+            Circuit: self
+
+        Raises:
+            IndexError: If `circuit` has no qubits.
```

Comment:
Should this be `self`?
```suggestion
            IndexError: If `self` has no qubits.
```